src/test/java/org/prebid/server/bidder/videobyte/VideobyteBidderTest.java

```diff
@@ -0,0 +1,286 @@
+                givenImp(identity()), givenImp(identity()));
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = bidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getValue()).hasSize(2)
+                .extracting(HttpRequest::getHeaders)
+                .flatExtracting(MultiMap::entries)
+                .hasSize(8)
+                .elements(2, 3, 6, 7)
+                .extracting(Map.Entry::getKey, Map.Entry::getValue)
+                .containsExactly(tuple(HttpUtil.ORIGIN_HEADER.toString(), "domain"),
+                        tuple(HttpUtil.REFERER_HEADER.toString(), "referer"),
+                        tuple(HttpUtil.ORIGIN_HEADER.toString(), "domain"),
+                        tuple(HttpUtil.REFERER_HEADER.toString(), "referer"));
+        assertThat(result.getErrors()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorIfResponseBodyCouldNotBeParsed() {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall("Incorrect body", null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors())
+                .hasSize(1)
+                .allSatisfy(error -> {
+                    assertThat(error.getType()).isEqualTo(BidderError.Type.bad_server_response);
+                    assertThat(error.getMessage()).isEqualTo("Bad server response.");
+                });
+    }
+
+    @Test
+    public void makeBidsShouldReturnEmptyListIfBidResponseIsNull() {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall("null", null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnEmptyListIfBidResponseSeatBidIsNull() {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall("{}", null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnBidsWithCorrectMediaType() throws JsonProcessingException {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall(
+                givenBidResponse("id1", "id2"),
+                givenBidRequest(givenImp(imp -> imp.id("id2")),
+                        givenImp(imp -> imp.id("id1").banner(Banner.builder().build()))));
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).hasSize(2)
```

Comment:
No need to check for sized since you are using containsExactly. Also, please check for similar occurences.